angel_system/eval/ptg_evaluate.py

```diff
@@ -74,17 +98,16 @@ def run_eval(args):
     # ============================
     # Plot
     # ============================
-    for label, ts_range_pairs in gt_label_to_ts_ranges.items():
-        if label in dets_label_to_ts_ranges:
-            plot_activity_confidence(label=label, gt_ranges=ts_range_pairs, det_ranges=dets_label_to_ts_ranges, output_dir=output_dir)
-        else:
-            log.warning(f"No detections found for \"{label}\"")
+    vis = EvalVisualization(labels=labels, gt=gt_label_to_ts_ranges, dets=dets_label_to_ts_ranges, output_dir=output_dir)
+    vis.plot_activities_confidence()
+    vis.plot_pr_curve()
```

Comment:
I presume we want the intersection threshold to be higher than 0.0 (which is currently the default parameter value for `plot_pr_curve)?